packages/master-detail-layout/test/stack-mode.test.js

```diff
@@ -52,7 +52,7 @@ describe('stack mode', () => {
       });
 
       it('should clear the stack mode when the layout size is bigger than stack threshold', async () => {
```

Comment:
Let's update the test name to reflect that this now depends on whether master and detail fit into the layout.

Reply:
Done.